docs/source/scheduler-overview.rst

```diff
@@ -0,0 +1,146 @@
+Scheduler Overview
+==================
+
+Once a dask graph is created, a scheduler is needed to run it. Dask currently
+implements a few different schedulers:
+
+- ``dask.threaded.get``: a scheduler backed by a thread pool
+- ``dask.multiprocessing.get``: a scheduler backed by a process pool
+- ``dask.async.get_sync``: a synchronous scheduler, good for debugging
+- ``dask.distributed``: a distributed scheduler for executing graphs on multiple machines
+
+
+The ``get`` function
+--------------------
+
+The entry point for all schedulers is a ``get`` function. This takes a dask
+graph, and a key or list of keys to compute.
+
+.. code-block:: python
+
+   >>> dsk = {'a': 1,
+   ...        'b': 2,
+   ...        'c': (add, 'a', 'b'),
+   ...        'd': (sum, ['a', 'b', 'c'])}
+
+   >>> get(dsk, 'c')
+   3
+
+   >>> get(dsk, 'd')
+   6
+
+   >>> get(dsk, ['a', 'b', 'c'])
+   [1, 2, 3]
+
+
+Using ``compute`` methods
+-------------------------
+
+Most of the time when working with dask collections you shouldn't need to
+interact with scheduler ``get`` functions directly. Each collection has a
+default scheduler, and a built-in ``compute`` method that calculates the output
+of the collection:
+
+.. code-block:: python
+
+    >>> import dask.array as da
+    >>> x = da.arange(100, chunks=10)
+    >>> x.sum().compute()
+    4950
+
+The compute method takes a number of keywords:
+
+- ``get``: a scheduler ``get`` function, overrides the default for the collection
+- ``**kwargs``: extra keywords to pass on to the scheduler ``get`` function (see :ref:`configuring-schedulers`)
+
+
+The ``compute`` function
+------------------------
+
+Sometimes you want to compute results from multiple dask collections at once.
+Similar to the ``compute`` method on each collection, there is a general
+``compute`` function that takes multiple collections and returns multiple
+results. This merges the graphs from each collection, so intermediate results
+are shared.
+
+.. code-block:: python
+
+    >>> y = (x + 1).sum()
+    >>> z = (x + 1).mean()
+    >>> da.compute(y, z)    # Compute y and z, sharing intermediate results
+    (5050, 50.5)
```

Comment:
This point might benefit from repetition.  

Perhaps expand on why this is good.  Particularly stating that we will only have to compute `x + 1` once.